src/components/card.js

```diff
@@ -14,10 +14,10 @@ class Card extends Component {
     const person = this.props.person;
     return (
       <div className="card">
-        <div className="person-title" id={person.get('person_id') + "-title"}>
+        <div className="person-title" id={person.getIn(['0','person_id']) + "-title"}>
           Who is this?
         </div>
-        <img src={person.get('image_url')}
+        <img src={person.getIn(['0','image_url'])}
```

Comment:
I think this will break in production, since the API does not yet return an array. Can you please remove this commit from the PR?